src/slices/GiveThunk.ts

```diff
@@ -73,7 +77,13 @@ export const changeApproval = createAsyncThunk(
       }
     }
 
-    const giveAllowance = await mockSohmContract._allowedValue(address, addresses[networkID].GIVING_ADDRESS);
+    let giveAllowance;
+    if (networkID === 1) {
+      giveAllowance = await sohmContract?.allowance(address, addresses[networkID].GIVING_ADDRESS);
+    } else if (networkID === 4) {
+      giveAllowance = await sohmContract?._allowedValue(address, addresses[networkID].GIVING_ADDRESS);
```

Comment:
Can you document why this requires a different function?

Reply:
yup!